include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -182,6 +182,31 @@ def LoopNormalize : Pass<"cc-loop-normalize"> {
   ];
 }
 
+def LoopPeeling : Pass<"cc-loop-peeling"> {
```

Comment:
For my own benefit, is this a per-function pass, or a module-level pass when it isn't specified?

Reply:
It defaults to module from the command-line.

I think the idea is that the pass manager could schedule it either way when you leave it as a generic `Operation*`.